docs/source/en/api/pipelines/flux.md

```diff
@@ -160,18 +162,115 @@ image.save("flux-fp8-dev.png")
 
 ## FluxPipeline
 
-[[autodoc]] FluxPipeline
-	- all
-	- __call__
+[[autodoc]] FluxPipeline - all - **call**
 
 ## FluxImg2ImgPipeline
 
-[[autodoc]] FluxImg2ImgPipeline
-	- all
-	- __call__
+[[autodoc]] FluxImg2ImgPipeline - all - **call**
 
 ## FluxInpaintPipeline
 
-[[autodoc]] FluxInpaintPipeline
-	- all
-	- __call__
+[[autodoc]] FluxInpaintPipeline - all - **call**
+
+## Flux ControlNet Inpaint Pipeline
```

Comment:
We don't need to provide an example here. autodoc will take the example from the `EXAMPLE_DOC_STRING` defined at the top of the pipeline file here and show the relevant usage.

Reply:
sure